content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. This makes the bugzilla plugin run a couple
+of checks:
+* If a bug with the id `123` exists and is in state `open`
```

Comment:
is this superceded by the "correct state" configuration below?

Reply:
No, `is_open` is a boolean bug attribute, my best guess is that a bug is open if it is in any state that is not `Closed`. The rest api reference isn't really very helpful on that either:
```
is_open | boolean | true if this bug is open, false if it is closed.
```

Ref:  https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#get-bug

Reply:
weird.  So you can both set the "is_open: true" field *and* "valid_state: closed" ?

guessing is_open predates valid_state.  Or am I misunderstanding how valid_state is used?

Reply:
Also should this instead say "if a bug with the id `123` exists and is in the required open or closed state as configured by is_open [link to example config]"  ?

since just like everything else, it's configurable on a per repo basis.  Maybe mention our default is `open` but ultimately you are straddling a line between documenting what the plugin allows to be configured, and how our repositories are configured (and how our repositories are configured by default vs overridden specifically).

Reply:
> weird. So you can both set the "is_open: true" field and "valid_state: closed" ?

Technically yes, but a bug will never be able to satisfy both requirements. It will however be able to satisfy something like `is_open: true` and `valid_state: New`